src/actions/projectsActions.js

```diff
@@ -481,19 +481,7 @@ export const editProjectACreator = (
             };
 
             const overViewKeys = {
-              keys: cutNotNeededKeysFromArray(Object.keys(getProjectResponse), [
-                0,
-                1,
-                2,
-                ,
-                4,
-                8,
-                9,
-                10,
-                11,
-                12,
-                13
-              ]),
```

Comment:
What was that? 🗡 xD

Reply:
They are indexes of project array, which should be avoid.
![projdetails](https://user-images.githubusercontent.com/22744668/49521594-15df1080-f8a6-11e8-9cfe-2ff343819663.png)